docs/test/coverage.md

```diff
@@ -26,32 +25,36 @@ All files    |   38.89 |   42.11 |
 -------------|---------|---------|-------------------
 ```
 
-If coverage is below a threshold, `bun:test` will exit with a non-zero exit code to indicate the failure.
-
-### Configuring coverage
-
-`bunfig.toml` supports configuring coverage:
+To always enable coverage reporting by default, add the following line to your `bunfig.toml`:
 
 ```toml
 [test]
 
-# Always enable coverage
+# always enable coverage
 coverage = true
+```
 
-# Anything less than 90% coverage will fail the test
-# coverageThreshold = 0.9
-coverageThreshold = { line = 0.9, function = 0.9 }
+By default coverage reports will _include_ test files and _exclude_ sourcemaps. This is usually what you want, but it can be configured otherwise in `bunfig.toml`.
```

Comment:
The part about sourcemaps isn't quite accurate. I suggest explaining them separately.

Something like

> Internaly, Bun transpiles all files by default, which means that code coverage reporting needs to be sourcemapped to the original source code for line numbers to match. Bun does this automatically for you. If for any reason you want to disable this, set `coverageIgnoreSourcemaps` to `false`.